src/operator/tensor/init_op.h

```diff
@@ -174,6 +174,40 @@ struct RangeParam : public dmlc::Parameter<RangeParam> {
   }
 };
 
+struct RangeLikeParam : public dmlc::Parameter<RangeLikeParam> {
+  double start;
+  double step;
+  int repeat;
+  std::string ctx;
+  int dtype;
+  dmlc::optional<int> axis;
+
+  DMLC_DECLARE_PARAMETER(RangeLikeParam) {
+    DMLC_DECLARE_FIELD(start)
+    .set_default(0)
+    .describe("Start of interval. The interval includes this value. The default start value is 0.");
+    DMLC_DECLARE_FIELD(step)
+    .set_default(1)
+    .describe("Spacing between values.");
+    DMLC_DECLARE_FIELD(repeat)
+    .set_default(1)
+    .describe("The repeating time of all elements."
+              " E.g repeat=3, the element a will be repeated three times --> a, a, a.");
```

Comment:
want to know how the `repeat` parameter work with:
https://github.com/apache/incubator-mxnet/blob/35db8e5a4719335a8974f2a8198a44e814fedaa0/src/operator/tensor/init_op.h#L494-L499
 and there is not repeat parameter on the `numpy.arange()`.

Reply:
Unfortunately `nd.arange` has a different interface compard to numpy.arange. I think it's reasonable to let `contrib.arange_like` have an interface similar to `nd.arange`. 
If we want to remove the repeat argument, it's best when we add the arange op to the `mx.np` namespace in the numpy project. 

@haojin2 @reminisce  FYI the nd.arange operator interface is inconsistent with numpy